components/Toastr/toastr.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <meta
+      name="description"
+      content="Toastr is a very nice component, and you can show success, error, warning & Informative messages."
+    />
+    <link rel="stylesheet" href="./../../styles/main.css" />
```

Comment:
from your folder structure you can do '/styles/main.css'
instead of this '../../' paths
You can make that change same in other files

Reply:
Done